README.md

```diff
@@ -34,6 +34,7 @@ A cross platform plugin for displaying local notifications.
     * Big text
     * Inbox
     * Messaging
+    * Media
```

Comment:
I think a note should be left that it can only show pictures as album artwork i.e. no support for controlling media playback